### PR TITLE
feat(internal): Switch to duration and rename 'timeout'

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

File: internal/cli/kraft/cloud/instance/stop/stop.go [hunk 1]
```diff
@@ -21,11 +22,11 @@ import (
 )
 
 type StopOptions struct {
-	TimeoutMS int64  `local:"true" long:"timeout-ms" short:"w" usage:"Timeout for the instance to stop"`
-	Output    string `long:"output" short:"o" usage:"Set output format. Options: table,yaml,json,list" default:"table"`
-	All       bool   `long:"all" usage:"Stop all instances"`
-	Metro     string `noattribute:"true"`
-	Token     string `noattribute:"true"`
+	DrainTimeout time.Duration `local:"true" long:"drain-timeout" short:"d" usage:"Timeout for the instance to stop (ms/s/m/h)"`
```
Comment (antoineco) [inline internal/cli/kraft/cloud/instance/stop/stop.go:25]: "d" is not very good at conveying the idea of a "timeout" IMO.
I would understand "w" for "wait", or if "d" was a bool for drain yes/no.

Is it relevant to change the flag in this PR?

Comment (craciunoiuc) [reply internal/cli/kraft/cloud/instance/stop/stop.go:25]: I could do 2 PRs, but @felipehuici asked for it be renamed to `drain-timeout`, and people were just using the word "drain" and variations everywhere so I just thought to switch to `-d`

Comment (craciunoiuc) [reply internal/cli/kraft/cloud/instance/stop/stop.go:25]: here: https://discord.com/channels/879723098881028167/1088815657673572445/1213061589264113664

Comment (craciunoiuc) [reply internal/cli/kraft/cloud/instance/stop/stop.go:25]: we could also use `-t` for timeout :seal:

**Review by antoineco — APPROVED** [review]

File: internal/cli/kraft/cloud/instance/stop/stop.go (hunk 1 above)
Comment (craciunoiuc) [reply internal/cli/kraft/cloud/instance/stop/stop.go:25]: I take it `-d` is fine :D

Comment (craciunoiuc) [reply internal/cli/kraft/cloud/instance/stop/stop.go:25]: Will merge now

**Pull request merged**